### PR TITLE
blackarch/issues/2819 master v1.2.0 PKGBUILD

### PR DESCRIPTION
**Pull request by dualfade**
pspy v1.2.0-master; New PKGBUILD

### CODE REVIEW

File: packages/pspy/PKGBUILD [hunk 1]
```diff
@@ -1,39 +1,55 @@
 # This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
 # See COPYING for license details.
 
+# ArchLinux go guidelines --
+# https://wiki.archlinux.org/index.php/Go_package_guidelines
+
 pkgname=pspy
 pkgver=159.2312eed
 pkgrel=1
```
Comment (noraj) [inline packages/pspy/PKGBUILD:9]: When you modify a package you need to increment pkgrel to avoid conflicts

File: packages/pspy/PKGBUILD [hunk 2]
```diff
@@ -1,39 +1,55 @@
...
 pkgdesc='Monitor linux processes without root permissions.'
 arch=('x86_64' 'aarch64')
 groups=('blackarch' 'blackarch-misc' 'blackarch-recon')
-url='https://github.com/DominicBreuker/pspy'
+url="https://github.com/DominicBreuker/$pkgname"
```
Comment (noraj) [inline packages/pspy/PKGBUILD:13]: I know it's tempting but we usually don't put variables in the ` url`

File: packages/pspy/PKGBUILD [hunk 3]
```diff
@@ -1,39 +1,55 @@
...
+
+    install -Dm 755 pspy "$pkgdir/usr/bin/$pkgname"
+    install -Dm 644 images/* -t "$pkgdir/usr/share/$pkgname/images/"
+    install -Dm 644 docker/Dockerfile.* -t "$pkgdir/usr/share/$pkgname/docker/"
+    install -Dm 644 docker/entrypoint-* -t "$pkgdir/usr/share/$pkgname/docker/"
+    install -Dm 644 docker/etc/cron.d/changepwds -t "$pkgdir/usr/share/$pkgname/docker/etc/crond.d/"
+    install -Dm 644 docker/root/scripts/password_reset.py -t "$pkgdir/usr/share/$pkgname/docker/root/scripts/"
+    install -Dm 644 docker/var/spool/cron/crontabs/root -t "$pkgdir/usr/share/$pkgname/docker/var/spool/cron/crontabs/"
+    install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" *.md
+    install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
```
Comment (noraj) [inline packages/pspy/PKGBUILD:54]: Don't copy license when it's a generic one already included in the licenses package, see https://wiki.archlinux.org/index.php/PKGBUILD#license

File: packages/pspy/PKGBUILD [hunk 4]
```diff
@@ -1,39 +1,55 @@
...
+source=("$url/archive/master.zip")
+sha256sums=('7774cc9df3bbf7def59b5ab6ca97b048c40ccd155f9bb0fe9d13ecd42c217725')
 
-pkgver() {
-  cd $pkgname
+prepare(){
+    mkdir -p build/
 
-  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+    VERSION="v1.2.0-10-g2312eed"
```
Comment (noraj) [inline packages/pspy/PKGBUILD:22]: You need to keep `echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)`, it's for automatic pkgver update. Don't hardcode the actual version in here.

**Review by noraj — CHANGES_REQUESTED** [review]

**Review by Edu4rdSHL — CHANGES_REQUESTED** [review]
Hey @dualfade, thank you. Please make the changes requested by @noraj when you can.

**Review by Edu4rdSHL — APPROVED** [review]
LGTM. Thank you very much @dualfade! Next time do not forge to add the package name to `lists/to-release`

**Pull request merged**